rpc/src/accounts.rs

```diff
@@ -84,7 +84,7 @@ impl From<SigningError> for Error {
 }
 
 fn get_data_dir() -> PathBuf {
-    [&env!("HOME"), ".sawtooth"].iter().collect()
+    [&env!("HOME"), ".sawtooth", "keys"].iter().collect()
```

Comment:
If you look at how the cli handles this, there are separate data and key directory functions. I would do the same thing here and create a new `get_key_dir()` function that returns the key path using the `get_data_dir()` funciton.

Reply:
Agree, but suggest we do that in a separate PR